### ISSUE TITLE
broken links for HPC-UGent documentation

### ISSUE THREAD

**Issue by boegel** [hpcugent/vsc_user_docs#139]
The `intro-HPC` links for Gent at http://hpcugent.github.io/vsc_user_docs/ are currently broken.

From the Travis log @ https://travis-ci.org/hpcugent/vsc_user_docs/builds/251441849:

```
(../sites/gent/macros.tex
../sites/gent/macros.tex:12: You can't use `macro parameter character #' in ver
tical mode.
l.12 #
       get these with ssh-keyscan gligar01.ugent.be > file
../sites/gent/macros.tex:12:  ==> Fatal error occurred, no output PDF file prod
uced!
Transcript written on intro-HPC-mac-gent.log.
Collected error summary (may duplicate other messages):
  pdflatex: Command for 'pdflatex' gave return code 1
      Refer to 'intro-HPC-mac-gent.log' for details
Latexmk: Use the -f option to force complete processing,
 unless error was exceeding maximum runs of latex/pdflatex.
```

caused by changes in #132

**Comment by boegel** [comment]
problem is fixed with #140, but this should have been reported by Travis...

@wpoely86 Thoughts on how to avoid this from happening in the future?

**Comment by boegel** [comment]
fixed in #143
